### PR TITLE
Recipe for libsamplerate

### PR DESCRIPTION
**Pull request by stefan-balke**

### CODE REVIEW

**Comment by stefan-balke** [comment]
The tests are faked now cause we run `make check` while building.

Is not a good idea in my opinion, but I wasn't able to run `make check` in the test environment.

Any thoughts here?

File: recipes/libsamplerate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
```
Comment (ocefpaf) [inline recipes/libsamplerate/meta.yaml]: These dependencies are not available on Windows.

File: recipes/libsamplerate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+test:
+  commands:
+    - true  # we perform testing in the build phase
```
Comment (ocefpaf) [inline recipes/libsamplerate/meta.yaml]: Can you test for a command or the existence of a library/file? I am not OK with this.

File: recipes/libsamplerate/bld.bat [hunk 3]
```diff
@@ -0,0 +1 @@
+Make.bat
```
Comment (ocefpaf) [inline recipes/libsamplerate/bld.bat]: Do you have this `Make.bat` file? I don't see it in your PR.

Comment (stefan-balke) [inline recipes/libsamplerate/bld.bat]: It's in the repository:

https://github.com/erikd/libsamplerate

Comment (ocefpaf) [inline recipes/libsamplerate/bld.bat]: Thanks! So it comes with the source.

**Comment by stefan-balke** [comment]
Fun fact:

> You need to get the pre-built Win32 binaries for libsndfile

Okay, Windows build need libsndfile...chicken or egg...

**Comment by ocefpaf** [comment]
> Okay, Windows build need libsndfile...chicken or egg...

Sometimes we do re-package binaries instead of building them. It is not ideal, but a possible solution in some corner cases. I would recommend you to get the *nix builds working, merge this, and then work on the Windows builds.

File: recipes/libsamplerate/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+requirements:
+  build:
+    - autoconf # [not win]
+    - automake # [not win]
+    - libtool # [not win]
+    - pkg-config # [not win]
+
+test:
+  commands:
+    - true  # we perform testing in the build phase
```
Comment (ocefpaf) [inline recipes/libsamplerate/meta.yaml]: @stefan-balke can you fix this one too?

File: recipes/libsamplerate/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+about:
+  home: http://www.mega-nerd.com/SRC/
+  license: Dual (GPL + commercial)
```
Comment (jakirkham) [inline recipes/libsamplerate/meta.yaml]: This license is pretty fun. Could we know what GPL this is? Also, what is the deal with the commercial license?

File: recipes/libsamplerate/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf # [not win]
+    - automake # [not win]
+    - libtool # [not win]
+    - pkg-config # [not win]
```
Comment (jakirkham) [inline recipes/libsamplerate/meta.yaml]: Could we have two spaces between selectors and packages? Also, could we get the selectors inline with each other?

File: recipes/libsamplerate/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: libsamplerate
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsamplerate.git
+  git_tag: 548c6761a0348a83c5f6ee68aa1581c855092195
```
Comment (jakirkham) [inline recipes/libsamplerate/meta.yaml]: Is there a way to not use git here?

**Comment by jakirkham** [comment]
Any thoughts on this?

**Comment by ocefpaf** [comment]
Closing as "stale."

**Pull request closed without merging**